packages/angular/docs/webpack-browser-examples.md

```diff
@@ -27,7 +27,7 @@ To use a custom webpack configuration when building your Angular application, ch
 
 ##### Incrementally Building your Application
 
-The executor supports incrementally building your Angular application by building the workspace libraries it depends on _(that have been marked as buildable)_ and then building your application using the built source of the libraries.
+The executor supports incrementally building your Angular application by building the workspace libraries they depend on _(that have been marked as buildable)_ and then building your application using the built source of the libraries.
```

Comment:
This should stay `it depends`.

There is a single app depending on the libraries.